src/tests/integration/test_generic_db_opeations.py

```diff
@@ -140,3 +152,70 @@ def test_telemetry_execute_command_has_connection_info(
             },
         },
     )
+
+
+BOX_PLOT_FAIL_REASON = (
+    "Known issue, the SQL engine must support percentile_disc() SQL clause"
+)
+
+
+@pytest.mark.parametrize(
+    "cell",
+    [
+        ("%sqlplot histogram --table plot_something --column x"),
+        ("%sqlplot hist --table plot_something --column x"),
+        ("%sqlplot histogram --table plot_something --column x --bins 10"),
+        pytest.param(
+            "%sqlplot boxplot --table plot_something --column x",
+            marks=pytest.mark.xfail(reason=BOX_PLOT_FAIL_REASON),
```

Comment:
this is xfailing the tests with all databases right? can just xfail with MySQL? (I think all the others have percentile_disc)

Reply:
Fix by separating `boxplot` and `histogram` test case

Reply:
perhaps I'm missing something but seems like we're still marking xfail for all databases? we should check that we can create the boxplot with every database that has the percentile disc, AFAIK the only one that doesn't implement it is MySQL